dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/tasks/CreateBucketTask.java

```diff
@@ -53,9 +55,14 @@ protected Object call() throws Exception {
         try {
             return ds3Client.putBucketSpectraS3(new PutBucketSpectraS3Request(bucketName)
                     .withDataPolicyId(createBucketModel.getId()));
-        } catch (final Exception e) {
+        } catch (final FailedRequestException fRE) {
+            final Joiner joiner = Joiner.on(", ");
+            LOG.error("Failed to create bucket", fRE);
+            loggingService.logMessage("While creating bucket expected status code " + joiner.join(fRE.getExpectedStatusCodes()) + " but got " + fRE.getStatusCode(), LogType.ERROR);
```

Comment:
Is there a way we can get a portion of the error message to determine what went wrong?  Or if not the whole message, determine which one it is and display our own custom error message?

Reply:
I agree, perhaps we should make this less verbose for now, and work on a translation between BP Errors and User errors?

Reply:
That seems like a reasonable approach.